Doc/library/webbrowser.rst

```diff
@@ -40,14 +40,23 @@ a new tab, with the browser being brought to the foreground. The use of the
 :mod:`webbrowser` module on iOS requires the :mod:`ctypes` module. If
 :mod:`ctypes` isn't available, calls to :func:`.open` will fail.
 
+.. program:: webbrowser
+
 The script :program:`webbrowser` can be used as a command-line interface for the
 module. It accepts a URL as the argument. It accepts the following optional
 parameters:
 
-* ``-n``/``--new-window`` opens the URL in a new browser window, if possible.
-* ``-t``/``--new-tab`` opens the URL in a new browser page ("tab").
+.. option:: -n, --new-window
+
+   Opens the URL in a new browser window, if possible.
+
+.. option:: -t, --new-tab
+
+   Opens the URL in a new browser page ("tab").
```

Comment:
I think we can update this too, browser tabs are now well established compared with when this feature was introduced in 2005: https://github.com/python/cpython/commit/e8f244305ef4

(Opera had tabs in 2000, Safari in 2003, IE7 in 2006: https://en.wikipedia.org/wiki/Tab_(interface)#History)

```suggestion
   Opens the URL in a new browser tab.
`